post/templates/post/av.html

```diff
@@ -33,7 +33,7 @@ <h3 class="panel-title">
         </div>
 
         {% if not category.counting %}
-          <table class="table table-condensed">
+          <table class="table table-sm">
```

Comment:
Is this new in bootstrap4? :o the `table-condensed` class was something we introduced in our own bootstrap3 theme.

Reply:
as far as i know, `table-condensed` has been in bootstrap for a long time. https://getbootstrap.com/docs/3.3/css/#tables-condensed

Reply:
Ah you're right. I had it confused with a custom [`form-condensed`](https://github.com/WISVCH/bootstrap-theme/blob/gh-pages/sass/wisvch-form-condensed.scss)